### PR TITLE
Add more distros to mrack config

### PR DESCRIPTION
**Pull request by lukaszachy**
Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: tmt/steps/provision/mrack/mrack-provisioning-config.yaml [hunk 1]
```diff
@@ -29,6 +30,10 @@ beaker:
         rhel-8.7-nightly: RHEL-8.7%
         rhel-8.8: RHEL-8.8%
         rhel-8.8-nightly: RHEL-8.8%
+        rhel-8.9: RHEL-8.9%
+        rhel-8.9nightly: RHEL-8.9%
```
Comment (happz) [inline tmt/steps/provision/mrack/mrack-provisioning-config.yaml:34]: Missing `-`?

Comment (dav-pascual) [reply tmt/steps/provision/mrack/mrack-provisioning-config.yaml:34]: Yup

Comment (lukaszachy) [reply tmt/steps/provision/mrack/mrack-provisioning-config.yaml:34]: Thanks, reviews ate good thing

**Review by psss — APPROVED** [review]
Thanks for the update!

**Pull request merged**